### PR TITLE
new parser ls_edac_mc

### PR DESCRIPTION
**Pull request by rasrivas-redhat**
Signed-off-by: Rahul Srivastava rasrivas@redhat.com

### CODE REVIEW

**Comment by bfahr** [comment]
@rasrivas-redhat please make sure all of your commits use the `-s` switch to "sign off" on your commits.  See [this reference](https://probot.github.io/apps/dco/) for more information.

File: insights/parsers/ls_edac_mc.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 power
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc0
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc1
+    drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc2
+
+Examples:
+    >>> '/sys/devices/system/edac/mc' in ls_edac_mc
+    True
+    >>> ls_edac_mc.dirs_of('/sys/devices/system/edac/mc') == ['.', '..', 'power', 'mc0', 'mc1', 'mc2']
+    True
```
Comment (bfahr) [inline insights/parsers/ls_edac_mc.py]: Please follow the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) to include documentation for your parser in the build process.  Make sure that you test and review your documentation by running `make clean html_debug` in your docs directory before committing changes.  You are also missing the file `docs/shared_parsers_catalog/ls_edac_mc.rst` to include your parser docs in the build.

File: insights/parsers/tests/test_ls_edac_mc.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc0
+drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc1
+drwxr-xr-x. 2 0 0 0 Jan 10 10:33 mc2
+"""
+
+
+def test_ls_edac_mc():
+    ls_edac_mc = LsEdacMC(context_wrap(LS_EDAC_MC))
+    assert '/sys/devices/system/edac/mc/' in ls_edac_mc
+    assert ls_edac_mc.dirs_of('/sys/devices/system/edac/mc') == ['.', '..', 'power', 'mc0', 'mc1', 'mc2']
```
Comment (bfahr) [inline insights/parsers/tests/test_ls_edac_mc.py]: This test is failing, please take a look at the output from pytest by running the command `pytest` in the root dir of your project.

Comment (rasrivas-redhat) [reply insights/parsers/tests/test_ls_edac_mc.py]: @bfahr - ack, fixed the issue

File: insights/parsers/ls_edac_mc.py (hunk 1 above)
Comment (rasrivas-redhat) [reply insights/parsers/ls_edac_mc.py]: @bfahr - ack, fixed the issue

**Comment by rasrivas-redhat** [comment]
@bfahr - there is some issue with adding the signature if I add them as suggested by DOC error, there are couple of extra commits added to this PR.

**Comment by bfahr** [comment]
@rasrivas-redhat each of your commits must be "signedoff" to pass the DCO test.  If you click on the details for the DCO it provides some direction on how to fix the issue.

**Comment by rasrivas-redhat** [comment]
@bfahr - ack, the issue has been resolved, please check.

**Pull request merged**